pkgs/development/python-modules/psycopg2cffi/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, cffi, postgresql, openssl, six }:
+
+buildPythonPackage rec {
+  pname = "psycopg2cffi";
+  version = "2.8.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0yc0cxxkfr35kd959wsagxfhy6ikbix71rp9x0rmn858fxa4vapw";
+  };
+
+  nativeBuildInputs = [ postgresql ];
+  propagatedBuildInputs = [ six cffi ];
+
+  doCheck = false;
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Reply:
Found a fork that seems maintained and has tests that at least run. Trying that for now.

They do, however, fail because they require a postgres database running to test against. What do I do?

Reply:
In that case we disable the tests.

Reply:
Done, also added a note.

I did add the `pythonImportsCheck` at least.